src/app/cli/src/coda_run.ml

```diff
@@ -178,14 +178,20 @@ let setup_local_server ?(client_whitelist = []) ?rest_server_port ~coda
           let r = Coda_lib.request_work coda in
           Option.iter r ~f:(fun r ->
               Logger.trace logger ~module_:__MODULE__ ~location:__LOC__
-                !"Get_work: %{sexp:Snark_worker.Work.Spec.t}"
-                r ) ;
+                "responding to a Get_work request with some new work"
+                ~metadata:
+                  [ ( "work_spec"
+                    , `String (sprintf "%{sexp:Snark_worker.Work.Spec.t}" r) )
```

Comment:
```sprint !"%{sexp...}" r``` The exclamation is missing here and below. I think the order of arguments actually is irrelevant here...